src/content-tags-drawer/ContentTagsDrawer.tsx

```diff
@@ -83,40 +225,45 @@ TaxonomyList.propTypes = {
  * - If you want to use it as react component, you need to pass the content id and the close functions
  *   through the component parameters.
  */
-const ContentTagsDrawer = ({ id, onClose }) => {
+const ContentTagsDrawer = ({
+  id,
+  onClose,
+  variant = 'drawer',
+}: ContentTagsDrawerProps) => {
   const intl = useIntl();
   // TODO: We can delete 'params' when the iframe is no longer used on edx-platform
   const params = useParams();
   const contentId = id ?? params.contentId;
 
+  if (contentId === undefined) {
+    throw new Error('Error: contentId cannot be null.');
+  }
+
   const context = useContentTagsDrawerContext(contentId);
   const { blockingSheet } = useContext(ContentTagsDrawerSheetContext);
 
   const {
     showToastAfterSave,
     toReadMode,
     commitGlobalStagedTagsStatus,
-    isContentDataLoaded,
-    contentName,
     isTaxonomyListLoaded,
     isContentTaxonomyTagsLoaded,
     stagedContentTags,
     collapsibleStates,
-    isEditMode,
-    commitGlobalStagedTags,
-    toEditMode,
     toastMessage,
     closeToast,
     setCollapsibleToInitalState,
     otherTaxonomies,
   } = context;
 
-  let onCloseDrawer = onClose;
-  if (onCloseDrawer === undefined) {
+  let onCloseDrawer: () => void;
+  if (onClose === undefined) {
     onCloseDrawer = () => {
       // "*" allows communication with any origin
       window.parent.postMessage('closeManageTagsDrawer', '*');
```

Comment:
I think this is generating some warnings in the console. Maybe run this conditionally with the `drawer` variant, as it is the only one that could be in the iFrame?

```
Failed to execute 'postMessage' on 'DOMWindow': The target origin provided ('http://studio.local.edly.io:8001') does not match the recipient window's origin ('http://apps.local.edly.io:2001'). [lib:32131:1321312321](http://apps.local.edly.io:2001/course-authoring/library/lib:32131:1321312321)
```

Reply:
Updated: https://github.com/openedx/frontend-app-authoring/pull/1299/commits/1db25e826991fdf6e303fc169c2cb0fa11d95f84